staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -671,13 +734,40 @@ func (c *Cacher) processEvent(event *watchCacheEvent) {
 }
 
 func (c *Cacher) dispatchEvents() {
+	// Jitter to help level out any aggregate load.
+	bookmarkTimer := c.clock.NewTimer(wait.Jitter(time.Second, 0.25))
+	// Stop the timer when watchBookmarkFeatureGate is not enabled.
+	if !c.watchBookmarkEnabled && !bookmarkTimer.Stop() {
+		<-bookmarkTimer.C()
+	}
+	defer bookmarkTimer.Stop()
+
+	lastResourceVersion := uint64(0)
+	bookmarkEvent := &watchCacheEvent{
```

Comment:
It's not safe to pass the same object every time, given that you modify it below. You have to create a new object for every bookmark.

Reply:
done